ABI_BREAKS.md

```diff
@@ -6,4 +6,4 @@ This document lists the ABI breaks that were made in each mlibc major version.
 
 - [#452](https://github.com/managarm/mlibc/pull/452): The functions `FD_{CLR,ISSET,SET,ZERO}` were renamed to `__FD_{CLR,ISSET,SET,ZERO}` and replaced by macros to match Wine's assumptions.
 - [#511](https://github.com/managarm/mlibc/pull/511): Musl's regex engine was added, implementing `regcomp` and `regexec`. This required some changes to the `regex_t` struct.
-- [#504](https://github.com/managarm/mlibc/pull/504): A `domainname` member was added to `struct utsname`, which is a glibc extension.
+- [#580](https://github.com/managarm/mlibc/pull/504): A `domainname` member was added to `struct utsname`, which is a glibc extension.
```

Comment:
But the old entry is still true? We should just add two entries here.

Reply:
The old entry changed the Linux ABI, which isn't really something that downstream users will care about

Reply:
There should still be two entries. We cannot assume that no user is interested in the changelog of linux ABI changes.

Reply:
Fair point.